cmake/modules/OmrTargetSupport.cmake

```diff
@@ -108,7 +112,8 @@ function(omr_add_library name)
 	endif()
 
 	if(opt_SHARED)
-		# split debug info if applicable. Note: omr_split_debug is responsible for checking OMR_SEPARATE_DEBUG_INFO
+		# Split debug info if applicable. Note: omr_split_debug is responsible for checking OMR_SEPARATE_DEBUG_INFO, and
+		# _omr_toolchain_separate_debug_symbols is responsible for checking OMR_SKIP_STRIP.
```

Comment:
Please shorten these lines and correct the references to `omr_process_split_debug` (here and on line 204):
```
		# Split debug info if applicable. Note: omr_process_split_debug
		# is responsible for checking OMR_SEPARATE_DEBUG_INFO, and
		# _omr_toolchain_separate_debug_symbols is responsible for checking
		# OMR_SKIP_STRIP.
```